dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -124,6 +124,7 @@
     <Import Project="..\FlipView\FlipView.vcxitems" Label="Shared" />
     <Import Project="..\ComboBox\ComboBox.vcxitems" Label="Shared" />
     <Import Project="..\Pivot\Pivot.vcxitems" Label="Shared" />
+    <Import Project="..\ScrollBar\ScrollBar.vcxitems" Label="Shared" Condition="$(BuildLeanMuxForTheStoreApp) != 'true'" />
```

Comment:
Same question as the others about whether we need the BuildLeanMuxForTheStoreApp check

Reply:
We are no longer actively building the "lean" version any more. We can probably leave the "machinery" to build this variant in the repo in case we ever want to resurrect something similar. It doesn't hugely matter at the moment which set of projects are in or out of the lean build, since we would want to re-evaluate that set at the time we brought it back. So I would say, for now just leave the Condition off. But including it is not doing any harm either.